lib/knapsack_pro/test_case_detectors/rspec_test_example_detector.rb

```diff
@@ -33,32 +33,22 @@ def generate_json_report(rspec_args)
         # Apply a --format option which overrides formatters from the RSpec custom option files like `.rspec`.
         cli_args = cli_args_without_formatters + cli_format + [
           '--dry-run',
-          '--no-color',
           '--out', report_path,
-          '--default-path', test_dir,
+          '--default-path', test_dir
         ] + KnapsackPro::TestFilePresenter.paths(test_file_entities)
-        options = ::RSpec::Core::ConfigurationOptions.new(cli_args)
-        exit_code = ::RSpec::Core::Runner.new(options).run($stderr, $stdout)
-        if exit_code != 0
-          debug_cmd = ([
-            'bundle exec rspec',
-          ] + cli_args).join(' ')
-
-          KnapsackPro.logger.error('-'*10 + ' START of actionable error message ' + '-'*50)
```

Comment:
suggestion

Since this is removed, we need to update the example in docs: https://docs.knapsackpro.com/ruby/split-by-test-examples/#how-do-i-fix-could-not-generate-json-report-for-rspec-rake-task-failed-when-running-rack_envtest-rails_envtest-bundle-exec-rake-knapsack_prorspec_test_example_detector